doc/pages/example_notebooks/Diffusion/AdiabaticCooling.ipynb

```diff
@@ -27,13 +27,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%matplotlib notebook\n",
+    "%matplotlib inline\n",
     "\n",
     "import pandas as pd\n",
     "import matplotlib.pyplot as plt\n",
     "import seaborn as sns\n",
     "import numpy as np\n",
     "from scipy.optimize import curve_fit\n",
+    "import sys\n",
+    "sys.path.append('../../../../build/')\n",
```

Comment:
Is this necessary?

Reply:
No, this is not necessary and probably also bad practice. I run the examples locally in a wrong environment and forgot to remove the import before pushing.